pyvista/utilities/reader.py

```diff
@@ -1295,6 +1320,68 @@ def auto_detect_format(self):
     def auto_detect_format(self, value):
         self.reader.SetAutoDetectFormat(value)
 
+    def add_function(self, value: Union[int, Plot3DFunctionEnum]):
+        """Specify additional functions to read.
+
+        Parameters
+        ----------
+        value
```

Comment:
I think we need a description of the mapping from int to Enum, or a direct link to the Enum documentation which might suffice.